src/server/saved_objects/routes/bulk_get.js

```diff
@@ -0,0 +1,20 @@
+import Joi from 'joi';
+
+export const createBulkGetRoute = (prereqs) => ({
+  path: '/api/saved_objects/_bulk_get',
```

Comment:
Will do - the underscore can actually be removed. I had it simply to not conflict with any saved object key names, but that won't be an issue when we move to ID's.